pkg/agent/route/interfaces.go

```diff
@@ -44,4 +44,7 @@ type Interface interface {
 	// UnMigrateRoutesFromGw should move routes back from local gateway to original device linkName
 	// if linkName is nil, it should remove the routes.
 	UnMigrateRoutesFromGw(route *net.IPNet, linkName string) error
+
+	// Run periodically syncs required firewall rules.
```

Comment:
firewall rules are not very suitable for what it actually does.
Maybe we don't need to be specific and could just say "Run starts the sync loop."